ctru-rs/src/applets/mii_selector.rs

```diff
@@ -56,14 +56,16 @@ pub struct MiiSelector {
 #[derive(Clone, Debug)]
 pub struct SelectionResult {
     pub mii_data: MiiData,
-    pub is_mii_selected: bool,
     pub mii_type: MiiType,
 }
 
 /// Error type for the Mii selector
 #[derive(Copy, Clone, Debug, Eq, PartialEq)]
```

Comment:
doesn't need to be in this PR but we probably should make this `#[non_exhaustive]` as well